satpy/tests/writer_tests/test_cf.py

```diff
@@ -63,7 +63,7 @@ def test_save_array(self):
                                    "resolution=None, polarization=None, "
                                    "calibration=None, level=None, modifiers=())")
                 self.assertEqual(f['test-array'].attrs['prerequisites'][0],
-                                 np.string_(expected_prereq))
+                                 expected_prereq)
```

Comment:
Why isn't the string call needed?

Reply:
The attribute is now a string, not bytes. You could also compare against `np.string_(expected_prereq).decode()`